Switch/Switch.js

```diff
@@ -63,9 +63,9 @@ const SwitchBase = kind({
 			animated: !noAnimation
 		}),
 		children: ({children, skin}) => {
-			if (children !== 'circle') return children;
```

Comment:
If I'm reading this code right, I think we need to refactor this a bit.

If we remove `children` from `defaultProps` (which I know will annoy @webOS101 a bit but seems necessary if the default is contingent on other props), the following will fulfill the requirements that we

* use `children` when set
* use `squarelarge` for "carbon" (since `square` doesn't exist and `stop` is wrong)
* use `circlelage` for everything else

```js
		children: ({children, skin}) => {
			if (children) return children;

			switch (skin) {
				case 'carbon': return 'squarelarge';
				default: return 'circle';
			}
		}
```

Agreed?

Reply:
I'm annoyed that you think I'd be annoyed! :P

Reply:
@ryanjduffy your proposal above makes sense to me. I'm in favor of that simplification.